jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/ArrayByteBufferPool.java

```diff
@@ -683,7 +684,7 @@ private Map<String, Object> toMap()
             @Override
             public String toString()
             {
-                return "capacity=%d,in-use=%d/%d,pooled/acquires/releases=%d/%d/%d(%.3f%%),non-pooled/evicts/removes=%d/%d/%d".formatted(
+                return String.format(Locale.ROOT, "capacity=%d,in-use=%d/%d,pooled/acquires/releases=%d/%d/%d(%.3f%%),non-pooled/evicts/removes=%d/%d/%d",
```

Comment:
I'm wondering if we should have a Jetty.LOCALE constant that defaults to Locale.ROOT, but that we allow to be initialised differently in case somebody really wants localised dumps/logs?

@joakime thoughts?

Reply:
hmmm AI thinks best practice is to not - localise logs etc. so I think directly using Locale.ROOT is fine.

Reply:
I'm good with hardcoded Locale usage.
I wonder tho, is Locale.ROOT or Locale.ENGLISH a better choice?
Whatever we choose, it should be reflected in the other PR #13417 too.

Locale.ROOT has an odd description, I need to look into it a bit more...
https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Locale.html#ROOT

Reply:
AI suggested ROOT